lib/text-formatters.js

```diff
@@ -4,6 +4,9 @@
  */
 'use strict';
 
+var moment = require('moment');
```

Comment:
I'd suggest to put this `require` call inside the `formatDate` method so you only incur the overhead of loading `moment` if it's really needed.

Reply:
When would avoiding that overhead be helpful?